frontend/admin/src/components/Room/RoomList.tsx

```diff
@@ -54,15 +54,18 @@ const RoomGrid = () => {
           <Card className={classes.card}>
             <CardContent className={classes.title}>
               <Typography variant="h5" component="h2" align="center">
-                {data[id].roomNumber}
+                {data[id].name}
               </Typography>
             </CardContent>
             <CardContent>
               <Typography variant="subtitle1" component="h2" align="center">
-                {data[id].city}
+                <p>Floor {data[id].floor} </p>
               </Typography>
               <Typography variant="subtitle1" component="h2" align="center">
-                {data[id].building}
```

Comment:
There should still be a city and a building displayed. Although they will need to be fetched as related data and I'm not sure how react-admin deals with this right now. Maybe we'll just use multiple queries or the backend will provide the related data with the same one.